microshift_running_apps/microshift-gitops.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-gitops"]
+= Automating application management with the GitOps controller
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-gitops
+
+toc::[]
+
+GitOps with Argo CD for {microshift-short} is a lightweight, optional add-on controller derived from the Red Hat OpenShift GitOps Operator. GitOps for {microshift-short} uses the command-line interface (CLI) of Argo CD to interact with the GitOps controller that acts as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+[id="microshift-gitops-can-do_{context}"]
+== What you can do with the GitOps agent
+By using the GitOps with Argo CD agent with {microshift-short}, you can leverage the following principles and tasks:
```

Comment:
```suggestion
By using the GitOps with Argo CD agent with {microshift-short}, you can leverage the following principles and tasks:
```

Reply:
Vale complains about "leverage", but I read the rest of the text and honestly, I can't come up with anything better than "leverage". Maybe "take advantage of"???

Reply:
Yea, "leverage" is very markety. I went with "utilize" and deleted "and tasks." It really is Git principles.